app/boards/arm/glove80/Kconfig.defconfig

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_GLOVE80_LH
+
+config BOARD
+    default "glove80 lh"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
+    default y
+
+endif # BOARD_GLOVE80_LH
+
+if BOARD_GLOVE80_RH
+
+config BOARD
+    default "glove80 rh"
+
+endif # BOARD_GLOVE80_RH
+
+if BOARD_GLOVE80_LH || BOARD_GLOVE80_RH
+
+config ZMK_SPLIT
+    default y
+
+config BT_CTLR
+    default BT
+
+config ZMK_BLE
+    default y
+
+config ZMK_USB
+    default y
+
```

Comment:
These will need to be moved to the `<board>_defconfig` files after the #1140 merge. See #2006 for the details.